### DISCUSSION TITLE
Global settings

### DISCUSSION THREAD

**Discussion by boromyr** [freerouting/freerouting#237] [Q&A]
How do I set my freerouting preferences globally? I placed the .rules and .par files in `C:\Users\PC\AppData\Local\freerouting\runtime\conf\` but to no avail.

**Comment by andrasfuchs** [comment]
Thanks for the feedback @boromyr!

We don't have this at the moment, but hopefully I can implement this (see #167) in the next version.

What settings would you set globally if you could?

**Comment by boromyr** [comment]
Ideally, all of them. It would be very functional if we could have the settings in a single JSON file as you suggested in #167 at the global level, and if a `freerouting.json` file is present in the project folder, it would override the global JSON file. This way, we would have default settings for any DSN file and local settings for specific DSN cases.
